test/helpers/localNetAccounts.ts

```diff
@@ -0,0 +1,192 @@
+      "0xc9b5b488586bf92ed1fe35a985b48b92392087e86da2011896c289e0010fc6bf",
+  },
+  {
+    address: "0x313f340305199bA942f5e4C04E56d2A74cf38246",
+    privateKey:
+      "0xe6776a7310afaffed6aeca2b54b1547d72dbfc9268ed05850584ddce53cf87a1",
+  },
+  {
+    address: "0x76C74B9f1AA5F345215BDf4aA6739a22E12687e7",
+    privateKey:
+      "0xb454e1649f031838a3b63b2fb693635266e048754f23cae6d9718250e3fb8905",
+  },
+  {
+    address: "0x932426283C37903F3aCA58ED5D7d63E905c6303B",
+    privateKey:
+      "0xb8849e63d7c25960af6eaff78fd82fe916b2c20cf569aaf4fa259c15faedd146",
+  },
+  {
+    address: "0x42323701Ab0A13c1E3e9a16CAb8BC58e542E2c9D",
+    privateKey:
+      "0x53513db9b03255c58b5f535e6d9e15bb3bfed583839094126b9a42ce2aa7469c",
+  },
+  {
+    address: "0x456D00fbde22d021a5bA78761A7EC568Af4930Fa",
+    privateKey:
+      "0x66486a3148467413a10cc8891b657bf092d307e066a08b833b892913607aede0",
+  },
+  {
+    address: "0x65CB4D7368C50E8a980eD646A2BaaA51d8c88086",
+    privateKey:
+      "0x5918ecc0f743222dee4ae4f2be17965e785435af6223ad3bdff80354d893f0c2",
+  },
+  {
+    address: "0x3eD993dE8847a604A6FA72A44EE6F140aCba64A8",
+    privateKey:
+      "0xb76b8ce771bfccf0167c3b2a51993e7687a4d8cbfb9ced61a98f601a772bda08",
+  },
+  {
+    address: "0x00fF5Fd1648bE1FEB36009B4b5fb5f8a2e5eF60d",
+    privateKey:
+      "0xcb448613322f0ae09bb111e6bfd5be93480f1ec521b062a614f9af025c8f1852",
+  },
+  {
+    address: "0xFF3Ffe08eCC9733a4d138cD99207707BBA25d9C7",
+    privateKey:
+      "0x147840cb64e7c4ae02917144897c37b521b859ac643bf55ec83444c11c3a8a30",
+  },
+  {
+    address: "0xd3F791b092250da6D2edb6077Fc7a175DDab8E0e",
+    privateKey:
+      "0x1a1bf9026a097f33ce1a51f5aa0c4102e4a1432c757d922200ef37df168ae504",
+  },
+  {
+    address: "0x9F5c16B53c363c73AF15F1dd5c1f68A6a4328637",
+    privateKey:
+      "0xbb3457514f768615c8bc4061c7e47f817c8a570c5c3537479639d4fad052a98a",
+  },
+  {
+    address: "0x6f10E8209296Ea9e556f80b0Ff545D8175F271d0",
+    privateKey:
+      "0xfbd9e512cc1b62db1ca689737c110afa9a3799e1bc04bf12c1c34ac39e0e2dd5",
+  },
+];
+
+async function processFile(
+  filePath: string,
+  processDataFunction: (data: string) => number,
+): Promise<number> {
+  try {
+    await access(filePath);
+  } catch {
+    await writeFile(filePath, "");
+  }
+
+  try {
+    const release = await lockfile.lock(filePath, {
+      retries: {
+        retries: 30,
+        minTimeout: 100,
+        maxTimeout: 500,
+      },
+    });
+
+    try {
+      const data = await readFile(filePath, "utf-8");
+      const processedData = processDataFunction(data);
+      await writeFile(filePath, processedData.toString());
+
+      return processedData;
+    } finally {
+      await release();
+    }
+  } catch (error) {
+    console.error("Error during working with a file:", error);
+    throw error;
+  }
+}
+
+function processData(data: string): number {
+  let index: number;
+
+  if (data === "") {
+    index = 0;
+  } else {
+    index = Number(data) + 1;
```

Comment:
looks a bit weird. Why +1?

Reply:
Probably you could use more meaningful function name bcs `processData` is too generic.